data/docs/instrumentation/angular.mdx

```diff
@@ -376,55 +376,97 @@ Enable CORS in the OTel Receiver. Inside `docker/clickhouse-setup/otel-collector
+
+const resource = Resource.default().merge(
+  new Resource({
+    [SemanticResourceAttributes.SERVICE_NAME]: '<service_name>',
+    [SemanticResourceAttributes.SERVICE_VERSION]: '0.1.0',
+  })
+);
+
+const provider = new WebTracerProvider({ resource });
+
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+
+provider.addSpanProcessor(
+  new BatchSpanProcessor(
+    new OTLPTraceExporter({
+      url: 'https://ingest.{REGION}.signoz.cloud:443/v1/traces',
+    })
+  )
+);
+
+provider.register({
+  propagator: new B3Propagator(),
+});
+
+registerInstrumentations({
+  instrumentations: [
+    getWebAutoInstrumentations({
+      '@opentelemetry/instrumentation-document-load': {},
+      '@opentelemetry/instrumentation-user-interaction': {},
+      '@opentelemetry/instrumentation-fetch': {
+        propagateTraceHeaderCorsUrls: /.+/,
       },
-      otelcolConfig: {
-        url: 'http://127.0.0.1:4318/v1/traces', // URL of opentelemetry collector
+      '@opentelemetry/instrumentation-xml-http-request': {
+        propagateTraceHeaderCorsUrls: /.+/,
       },
     }),
-    //Insert OtelCol exporter module
-    OtelColExporterModule,
-    //Insert propagator module
-    CompositePropagatorModule,
   ],
-  ...
-})
+});
 ```
 
 This config would be enough to get you up and running. For more tweaks refer to [this](https://github.com/jufab/opentelemetry-angular-interceptor#readme) detailed documentation of the instrumentation library.
 
+Again, `http://localhost:4318/v1/traces` is the default url for sending your tracing data. We are assuming you have installed SigNoz on your localhost. Based on your environment, you can update it accordingly. It should be in the following format:
+
+`http://<IP of SigNoz backend>:4318/v1/traces`
+
+**NOTE:** Remember to allow incoming requests to port 4318 of machine where SigNoz backend is hosted.
+
+## Conclusion
+
+OpenTelemetry is the future for setting up observability for cloud-native apps. It is backed by a huge community and covers a wide variety of technology and frameworks. Using OpenTelemetry, engineering teams can instrument polyglot and distributed applications and be assured about compatibility with a lot of technologies.
+
+SigNoz is an open-source observability tool that comes with a SaaS-like experience. You can check out SigNoz by visiting its GitHub repo 👇
 
+[![SigNoz GitHub repo](/img/blog/common/signoz_github.webp)](https://github.com/SigNoz/signoz)
 
-Facing difficulties with instrumenting your application? Check out this video tutorial 👇
+If you are someone who understands more from video, then you can watch the below video tutorial on the same with SigNoz.
 
 <YouTube id="g-I-v1FtMoM" mute={false} />
 
 ## Sample Angular App
 - [Sample Angular App](https://github.com/SigNoz/angular-otel-native) - contains `instrument.ts` file with placeholder for URL and Ingestion Details (For SigNoz Cloud)
 
```

Comment:
We should include this FAQs section after Sample Angular App section - https://signoz.io/docs/instrumentation/angular/#frequently-asked-questions

This might be helpful for users. Also in this FAQs section, we should mention that this is specifically for self-hosted SigNoz.

Reply:
Fixed!